.readthedocs.yaml

```
@@ -20,3 +20,6 @@ sphinx:
 python:
   install:
   - requirements: requirements-django42.txt
+  - method: pip
+    extra_requirements:
+      - docs
```

This might fix the readthedocs build (I followed https://docs.readthedocs.io/en/stable/config-file/v2.html#python)

```suggestion
  - method: pip
    path: .
    extra_requirements:
      - docs
```